magento/magento2
# Store longer X-Forwarded-For header

### Description (*)
Header X-Forwarded-For could contains multiple IPs
This PR allow to store longer values

For instance in my DB I had value like this:
`4x.xxx.xx.xx, 1xx.xxx.xx.xx, 10.`, where xxx - masked IP address of customer, while last IP address was just cut due to limit on the field in DB.

### Related Pull Requests
N/A

### Fixed Issues (if relevant)
Fixes #28693

### Manual testing scenarios (*)
1. Install Magento behind reverse proxy and Varnish
2. Go to website and place order
3. Go to admin > Sales > Orders
4. Navigate to your order, see data in field "Placed from IP"

**Actual result:**
X-Forwarded-For header is cut

**Expected result:**
It shouldn't be cut

### Questions or comments
Ma

### Contribution checklist (*)
